conf/app.routes

```diff
@@ -10,12 +10,14 @@ GET        /unauthorised                                controllers.Unauthorised
 
 GET        /notifications                               controllers.NotificationsController.listOfNotifications()
 
-GET        /notifications/:mrn                          controllers.NotificationsController.listOfNotificationsForSubmission(mrn: String)
+GET        /notifications/:id                          controllers.NotificationsController.listOfNotificationsForSubmission(id: String)
```

Comment:
On backend we moved to `submissions/:id/notifications` maybe on frontend we should also

Reply:
Good suggestion

Reply:
Done